### ISSUE TITLE
Pinokio won't run on Windows 11 Pro Build 26002.1000

### ISSUE THREAD

**Issue by WilliamYowell** [pinokiocomputer/pinokio#70]
Windows 11 Pro insider preview Build 26002.1000
i7-13700KF
32 GB RAM
NVIDIA RTX 4070
more than 21TB hard drive capacity

Pinokio installs and starts, opens a window that remains blank with message "Pinoko (Not Responding)" in upper left corner.
Tried running in the compatibility mode. However, the newest mode available is Windows 8. Pinokio runs until I try to load anything, then an error message appears telling me that the current system is 9200, etc. Tried a virtual machine with Windows 10. Pinokio had error message that that virsion of Windows 10 is too old.

**Comment by luvis** [comment]
I see I'm not the only one. I've had this problem on the same build as @WilliamYowell . I'm now on build 26010 but the problem persists. I've tried uninstalling everything and removing all related files etc and reinstalling but it won't even reinstall now.

**Comment by WilliamYowell** [comment]
I am now up to Windows 11 Insider Preview build 26016, and still unable to run Pinokio. I have submitted a request to Microsoft to update compatibility modes, and I have not seen any response.

**Comment by WilliamYowell** [comment]
I just downloaded Pinokio 2.16, and it works in Windows 11 insider preview build 26016.

**Comment by feveromo** [comment]
Also having similar error while being on Windows 11 insider build 26020. Pinokio installs, opens, I browse the discover page and click download on something, new Pinokio windows opens then it all freezes.

**Comment by WilliamYowell** [comment]
Have you provided feedback to Microsoft to request a change to
compatibility mode. Pinokio will work with the commercial release of
Windows 11 (I am able to boot either commercial release or insider preview
on my computer). If I could select Windows 11 commercial release
compatibility, Pinokio should work.

**Comment by BleedingDev** [comment]
Same problem here - Install -> Run -> Discover -> Download model -> Freeze unresponsive window -> dies. :(

**Comment by Hollowiman** [comment]
Same problem here with W11 22H2 (22621.3155)

**Comment by luvis** [comment]
I just thought I should mention that I eventually decided to just reinstall my computer with win11 (non insider preview) and everything works like a charm. So it's indeed the Insider preview builds that have these issues. The Insider preview didn't exactly bring much to the table anyway and reinstalling, even if a bit of a hassle, wasn't too bad. So I'd just recommend anyone wanting to use pinokio to just bite that apple and reinstall. It's worth it.

**Comment by luvis** [comment]
> I updated my W11 Pro to > 22631.3155, same error (blank white screen after download buttons). And i dont have insider preview version.

I see, yeah then this might happen others as well. I guess my assumption about it only being insider preview related might be wrong then..I have the latest update too though without issues so it's at least not "global".. I guess we'll find out pretty soon if more people experience this.

**Comment by le-si** [comment]
I'm on the W11 Canary channel - crashes randomly,  sometimes crashes even after few seconds while browsing Discoveries. Not usable at all.

**Comment by kristianboserup** [comment]
It´s even worse here. Pinokio just crash on startup. Can´t even get to the discover page or anything else. I give up, tried reinstalling, but same story for months now

**Comment by shaal** [comment]
Same here. Starting Pinokio on Windows 11 Pro 22621.3296, the program loads with a plain blank window, and no way for me to interact with it.

**Comment by halr9000** [comment]
Known issue https://discord.com/channels/1121039057993089076/1223267322903658518

**Comment by darkanubis0100** [comment]
Same with Windows 11 Pro 24H2 26100.1000

**Comment by KhryptorGraphics** [comment]
yes, i have this issue as well. the discord details link does not work either

**Comment by SilverWulf212** [comment]
Killing me; same.  I'mrunning it at the office fine but won't wonk on my MSI system

**Comment by Kuchiriel** [comment]
Same here 

![Image](https://github.com/user-attachments/assets/3160790f-68b9-43b3-8dbd-9200564448c0)

![Image](https://github.com/user-attachments/assets/00b84b29-5406-42c6-8fd7-45176d7c3399)
